modules/media/jsx/mediaIndex.js

```diff
@@ -189,12 +190,12 @@ class MediaIndex extends React.Component {
   }
 }
 
-$(function() {
+window.onload = () => {
   const mediaIndex = (
     <div className="page-media">
-      <MediaIndex DataURL={`${loris.BaseURL}/media/?format=json`} />
+      <MediaIndex dataURL={`${loris.BaseURL}/media/?format=json`} />
     </div>
   );
 
   ReactDOM.render(mediaIndex, document.getElementById("lorisworkspace"));
-});
+};
```

Comment:
```suggestion
});
```

matches the change above.